docs/en-US/component/message-box.md

```diff
@@ -177,6 +177,8 @@ The corresponding methods are: `ElMessageBox`, `ElMessageBox.alert`, `ElMessageB
 | show-confirm-button          | whether to show a confirm button                                                                                                         | ^[boolean]                                                                         | true                                             |
 | cancel-button-text           | text content of cancel button                                                                                                            | ^[string]                                                                          | Cancel                                           |
 | confirm-button-text          | text content of confirm button                                                                                                           | ^[string]                                                                          | OK                                               |
+| cancel-button-loading-icon   | loading icon content of cancel button                                                                                                    | ^[string] / ^[Component]                                                           | Loading                                          |
+| confirm-button-loading-icon  | loading icon content of confirm button                                                                                                   | ^[string] / ^[Component]                                                           | Loading                                          |
```

Comment:
Please add a version number.

Reply:
I have added the version for new options, but ci fails for some unknown reason.🤔

Reply:
https://github.com/pnpm/action-setup/issues/135#issuecomment-2206861174